README.md

```diff
@@ -76,59 +88,64 @@ Here are the steps needed to run Terraform along with examples of each:
 
-![Virtual Machines](images/vsphere-virtual-machines.png)
+<img src="images/vsphere-virtual-machines.png" alt="Virtual Machines">
 
 ### Apache Web Servers
 
 Each Apache server has a custom `index.html` file that includes the hostname of the machine.
-
-![Apache Server Result](images/apache-server-result.png)
+<img src="images/apache-server-result.png" alt="Apache Server Result">
 
 ### AppDynamics Controller
 
-The 5 virtual machines also appear in the AppDynamics controller, each running an Apache Web Server, and all 5 appearing in the AppDynamics controller.
+The five virtual machines appear in the AppDynamics controller, each running an Apache Web Server, and all five appearing in the AppDynamics controller.
 
+<img src="images/appd-machine-agents.png" alt="List of Machine Agents">
 
+Click any of the check box available just before the `OS` column; then click `View Details` to see that the data reported by the Machine Agent to the AppDynamics Controller. 
 
-![List of Machine Agents](images/appd-machine-agents.png)
+You can see the data that is reported by the Machine Agent on `apache-webserver-1`.
 
-Click the check box next to any of the servers and then click `View Details` to see the data reported by the Machine Agent to the AppDynamics Controller. In the example below, we see data reported by the Machine Agent on `apache-webserver-1`.
+<img src="images/appd-web-server-1.png" alt="Data Reported by the Machine Agent">
 
-![Data Reported by the Machine Agent](images/appd-web-server-1.png)
 
 ### Extra Credit
 
-Now that you have an Apache Web Server running and you have a Machine Agent onboard your newly created hosts, you can monitor HTTP as a service. Here's how:
+Now that you have an Apache Web Server running and you have a Machine Agent onboarded your newly created hosts, you can monitor HTTP as a service. Here's how:
+
+1. Click `Servers` on the top navigation bar followed by `Service Availability` on the left-hand side of the AppDynamics controller user interface.
+2. Click `Add`.
+3. Enter a name for the service availability check (see the values we used in the example below).
+4. Enter a target address (a FQDN is needed to an A record in DNS is needed).
+5. Select the server that runs the check.  
+   In this case, `apache-web-server-2` is used to run a check against the HTTP service running on `apache-web-server-1`.
 
-- Click `Servers` on the top navigation bar followed by `Service Availability` on the left hand side of the AppDynamics controller user interface.
-- Click `Add`
-- Enter a name for the service availability check (see the values we used in the example below)
-- Enter a target address (a FQDN is needed to an A record in DNS is needed)
-- Select the server that is runs the check. In this case, we used `apache-web-server-2` to run a check against the HTTP service running on `apache-web-server-1`.
+<img src="images/add-service-monitoring-page-1.png" alt="HTTP Check">
```

Comment:
edit alt text "Select the server for an AppDynamics configuration for an HTTP Check"